assets/js/components/ChartSourceController/index.jsx

```diff
@@ -92,7 +92,8 @@ class ChartSourceController extends React.Component {
     }
 
     fetchActualExpenditureUrls(departmentName) {
-        let url = `../../actual-expenditure/?department_name=${encodeURI(departmentName)}`;
+        let selectedYear = document.querySelector('.YearSelect-item.is-active .YearSelect-link').innerText;
```

Comment:
don't use presentation stuff like page content inner text and css classes as data. Use data attributes if you want a reliable way to embed data.

There must be a variable in the page context which you can use for this.

Reply:
done